tests/e2e/tests/00-elasti-setup/02-assert.yaml

```
@@ -0,0 +1,32 @@
+  namespace: elasti
+status:
+  readyReplicas: 1
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: elasti-resolver
+  namespace: elasti
+status:
+  readyReplicas: 1
+--- 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)